### ISSUE TITLE
[FEMALE][3D Glasses]

### ISSUE THREAD

**Issue by LawPunks** [BlockNG-Foundation/LawPunks-Trek-BLOCKS#1]
## Status

- [x] Idea - Got some ideas so far.
- [ ] Draft - Already working on my plan.
- [ ] Review - PR has been created.
- [ ] Final - PR has been merged.
- [ ] Living - Already on board.

- Estimated time of complete: 2023-04-01
- Created: 2023-03-20

## Copyright

- The submitter guarantees that the submitted image is an original work and does not infringe the copyright or other rights of others. 
- Once the image is merged into the LawPunks project, its copyright will belong to BlockNG. Any use or redistribution of this image must be licensed by BlockNG. 
- BlockNG reserves the right to pursue legal action for any copyright infringement. 
- Thank you for your support of the LawPunks project.
- [ ] Accept
